src/main/java/com/williamfiset/algorithms/datastructures/stack/impl/ArrayStack.java

```diff
@@ -0,0 +1,62 @@
+package com.williamfiset.algorithms.datastructures.stack.impl;
+
+import com.williamfiset.algorithms.datastructures.stack.Stack;
+import java.util.Arrays;
+import java.util.EmptyStackException;
+
+/** @author liujingkun */
+public class ArrayStack<T> implements Stack<T> {
+  private int capacity;
+  private Object[] data;
+  private int size;
+
+  public ArrayStack() {
+    // the factor of 10 is the magic of speed!!
```

Comment:
Why 10? Add technical details instead of saying it's magic, or just say randomly chosen small number

Reply:
has changed to 16. it can be any small number.